File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/AmPm.java

package com.dinodevs.greatfitwatchface.theme.bin;

import java.util.HashMap;
import java.util.Map;

public class AmPm {

    private Integer x;
    private Integer y;
    private Integer imageIndexAMCN;
    private Integer imageIndexPMCN;
    private Integer imageIndexAMEN;
    private Integer imageIndexPMEN;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    public Integer getX() {
        return x;
    }

    public void setX(Integer x) {
        this.x = x;
    }

    public Integer getY() {
        return y;
    }

    public void setY(Integer y) {
        this.y = y;
    }

    public Integer getImageIndexAMCN() {
        return imageIndexAMCN;
    }

    public void setImageIndexAMCN(Integer imageIndexAMCN) {
        this.imageIndexAMCN = imageIndexAMCN;
    }

    public Integer getImageIndexPMCN() {
        return imageIndexPMCN;
    }

    public void setImageIndexPMCN(Integer imageIndexPMCN) {
        this.imageIndexPMCN = imageIndexPMCN;
    }

    public Integer getImageIndexAMEN() {
        return imageIndexAMEN;
    }

    public void setImageIndexAMEN(Integer imageIndexAMEN) {
        this.imageIndexAMEN = imageIndexAMEN;
    }

    public Integer getImageIndexPMEN() {
        return imageIndexPMEN;
    }

    public void setImageIndexPMEN(Integer imageIndexPMEN) {
        this.imageIndexPMEN = imageIndexPMEN;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/AmPm.kt
package com.dinodevs.greatfitwatchface.theme.bin

import java.util.*

class AmPm {
    var x: Int? = null
    var y: Int? = null
    var imageIndexAMCN: Int? = null
    var imageIndexPMCN: Int? = null
    var imageIndexAMEN: Int? = null
    var imageIndexPMEN: Int? = null
    private val additionalProperties: MutableMap<String, Any> = HashMap()

    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}